pkgs/servers/gotify/mod-sha.nix

```diff
@@ -1 +1 @@
-"0zpdbj8a0akhzi1ian8zs6y7ymhlpcfy13q64xdlwc5w0286554r"
\ No newline at end of file
+"1ha0zgz1n07sd3if6823fi83j7yajysjdzfbwqk9rpsi6zv3cfh3"
```

Comment:
Is this still used? Otherwise it can be deleted as well.

Reply:
Done.